src/components/settings/SettingsSSOTab/steps/SSOConfigConfigureStep.jsx

```diff
@@ -0,0 +1,113 @@
+import { Form, Hyperlink } from '@edx/paragon';
+import PropTypes from 'prop-types';
+import { HELP_CENTER_SAML_LINK } from '../../data/constants';
+
+const SSOFormControl = ({
+  label, floatingLabel, defaultValue, onChange, readOnly,
+}) => {
+  const labelNoSpaces = label.replace(/ /g, '');
+  return (
+    <>
+      <Form.Control
+        readOnly={readOnly}
+        className="mt-4 mb-1"
+        type="text"
+        onChange={onChange}
+        floatingLabel={floatingLabel || label}
+        defaultValue={defaultValue}
+        aria-labelledby={labelNoSpaces}
+      />
+      <Form.Label id={`sso-field-${labelNoSpaces}`} className="mb-2">
+        {label}
+      </Form.Label>
+    </>
+  );
+};
+
+SSOFormControl.defaultProps = {
+  floatingLabel: '',
+  readOnly: false,
+  defaultValue: undefined,
+  onChange: () => {},
+};
+
+SSOFormControl.propTypes = {
+  label: PropTypes.string.isRequired,
+  floatingLabel: PropTypes.string,
+  defaultValue: PropTypes.string,
+  onChange: PropTypes.func,
+  readOnly: PropTypes.bool,
+};
+
+const SSOConfigConfigureStep = () => {
+  const data = {
+    userId: '',
+    fullName: '',
+    firstName: '',
+    maxSessionLength: undefined,
+    emailAddress: '',
+    samlConfig: '',
+  };
+  // TODO: This is not yet implemented, will be in future work when we work on this screen.
```

Comment:
is this going implemented in a fast follow?

Reply:
ah it's a separate ticket already yah. I just left comments here in case I don't get to it myself before others do.